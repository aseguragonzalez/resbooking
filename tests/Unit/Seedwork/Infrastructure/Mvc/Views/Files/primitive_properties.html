<ul>
    <li>name: {{model->name}}</li>
    <li>age: {{model->age}}</li>
    <li>height: {{model->height}}</li>
    <li>isStudent(true): {{model->isStudent}}</li>
    <li>isEmployed(false): {{model->isEmployed}}</li>
    <li>createdAt: {{model->createdAt}}</li>
</ul>
