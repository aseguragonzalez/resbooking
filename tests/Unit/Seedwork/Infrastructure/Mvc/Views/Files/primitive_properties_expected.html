<ul>
    <li>name: Peter Parker</li>
    <li>age: 25</li>
    <li>height: 1.75</li>
    <li>isStudent(true): true</li>
    <li>isEmployed(false): false</li>
    <li>createdAt: +2025-01-02T12:01:02+00:00</li>
</ul>
